weave/ops_domain/table.py

```diff
@@ -676,9 +676,12 @@ def _get_table_data_from_file(file: artifact_fs.FilesystemArtifactFile) -> dict:
     tracer = engine_trace.tracer()
     if file is None or isinstance(file, artifact_fs.FilesystemArtifactDir):
         raise errors.WeaveInternalError("File is None or a directory")
+    print("before open", flush=True)
```

Comment:
dev tracks (ok not going to comment on any more of these)

Reply:
yeah sorry, i haven't removed these yet, in case we change the solution again. wanted to confirm if im on the right path now.